.github/workflows/node.js.yml

```diff
@@ -26,4 +26,5 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
     - run: npm run build --if-present
+    - run: npm run build-storybook
```

Comment:
https://github.com/hikariumeno/nuxt.js-practice/pull/3/files#r499219364

OK, thank you :)